internal/devicescale/impl_notx.go

```diff
@@ -12,8 +12,15 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !android && !ios && !js && (darwin || windows)
+// +build !android
+// +build !ios
+// +build !js
+// +build darwin windows
```

Comment:
`!android` and `!js` is not needed

Reply:
Same here - put it this way to make it more logical. Adjusted for your comments anyway.